### PR TITLE
fabric-crashlytics-deployer 1.0.1

### PR DESCRIPTION
**Pull request by manuelgon47**
### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [x] I read the [Step Development Guideline](https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md)
- [x] I have a test for my Step, which can be run with `bitrise run test` (in the step's repository)
- [x] I did run `bitrise run audit-this-step` (in the step's repository - note, if you don't have this workflow in your `bitrise.yml`, [you can copy it from the step template](https://github.com/bitrise-steplib/step-template/blob/master/bitrise.yml).)
- [x] I read and accept the [Abandoned Step policy](https://github.com/bitrise-io/bitrise-steplib#abandoned-step-policy)

### CODE REVIEW

File: steps/fabric-crashlytics-deployer/1.0.1/step.yml [hunk 1]
```diff
@@ -0,0 +1,84 @@
+title: Fabric / Crashlytics deployer
+summary: |
+  Deploy to Fabric/Crashlytics beta testing service your apk.
+description: |
+  Deploy to Fabric/Crashlytics beta testing service your apk.
+website: https://github.com/manuelgon47/bitrise-step-fabric---crashlytics-deployer
```
Comment (trapacska) [inline steps/fabric-crashlytics-deployer/1.0.1/step.yml]: This url points to a 404 page

File: steps/fabric-crashlytics-deployer/1.0.1/step.yml [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+source_code_url: https://github.com/manuelgon47/bitrise-step-fabric---crashlytics-deployer
```
Comment (trapacska) [inline steps/fabric-crashlytics-deployer/1.0.1/step.yml]: This url points to a 404 page

File: steps/fabric-crashlytics-deployer/1.0.1/step.yml [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+support_url: https://github.com/manuelgon47/bitrise-step-fabric---crashlytics-deployer/issues
```
Comment (trapacska) [inline steps/fabric-crashlytics-deployer/1.0.1/step.yml]: This url points to a 404 page

**Comment by trapacska** [comment]
Sorry for not noticing previously, could you please fix those URLs, as this step won't be managed by Bitrise fully, so it is important for the users who will use the step if they have an issue to report to the correct place! 🙂

**Comment by manuelgon47** [comment]
Of course, sorry for the inconvenience

**Comment by manuelgon47** [comment]
I open another PR https://github.com/bitrise-io/bitrise-steplib/pull/1362

**Pull request closed without merging**